chainerx_cc/CMakeLists.txt

```diff
@@ -179,6 +179,12 @@ get_third_party(optional-lite)
 include_directories(${CMAKE_CURRENT_BINARY_DIR}/optional-lite/include)
 
 # Test
+
+# TODO(durswd): Remove it. It is hack for MSVC+LLVM
```

Comment:
Could you add more explanation why this hack is needed?

Reply:
I wrote it.